examples/langchain_multimodal_rag.ipynb

```diff
@@ -232,46 +244,47 @@
    "id": "15410af0-fa5c-4aed-b554-f2165c6f482e",
    "metadata": {},
    "source": [
-    "Next, we'll set our OpenAI API key and create a vector store to embed and store our text and table documents using OpenAI's embedding model"
+    "Next, we'll set our NVIDIA API key and create a vector store to embed and store our text and table documents using an embedding model from NVIDIA's API catalog"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 13,
    "id": "53957974-c688-4521-8c61-09f2649d5d53",
    "metadata": {},
    "outputs": [],
    "source": [
     "import os\n",
     "from langchain_chroma import Chroma\n",
-    "from langchain_openai import OpenAIEmbeddings\n",
+    "from langchain_nvidia_ai_endpoints import NVIDIAEmbeddings\n",
     "\n",
-    "# TODO: Add your OpenAI API key here\n",
-    "os.environ[\"OPENAI_API_KEY\"] = \"<YOUR_OPENAI_API_KEY>\"\n",
+    "# TODO: Add your NVIDIA API key here\n",
+    "os.environ[\"NVIDIA_API_KEY\"] = \"<YOUR_NVIDIA_API_KEY>\"\n",
     "\n",
-    "vectorstore = Chroma.from_documents(documents=(texts+tables), embedding=OpenAIEmbeddings())"
+    "embedding = NVIDIAEmbeddings()\n",
+    "vectorstore = Chroma.from_documents(documents=(texts+tables), embedding=embedding)"
```

Comment:
I thought the intent here is to switch to Milvus in this example, given that's what our deployment guides setup.

We could then link to the LangChain docs and say, optionally, LangChain supports the use of other vector stores.

Reply:
Yeah not sure why these changes didn't make it into my PR they're all as they should be on my end. I'll see if repushing solves it